src/frontend/context/SimuleringContext.tsx

```diff
@@ -98,17 +101,21 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
     };
     const erMigreringMedStoppISimulering = erMigreringFraInfotrygd && skalStoppeISimulering();
 
-    const erNegativeMånedsbeløpPåMaksEnKrone = (simPerioder: ISimuleringPeriode[]) =>
+    const erAvvikIMånedsbeløpPåMaksEnKronePerBarn = (
+        simPerioder: ISimuleringPeriode[],
+        antallBarn: number
+    ) =>
         simPerioder
             .filter(periode => erFør(kalenderDato(periode.fom), kalenderDato(mars2023)))
             .map(periode => periode.resultat || 0)
-            .every(beløp => beløp <= 0 && beløp >= -1);
+            .every(beløp => Math.abs(beløp) <= antallBarn);
 
-    const erMigreringMedFeilutbetalingInnenforBeløpsgrenser =
-        erMigreringFraInfotrygd &&
-        erFeilutbetaling &&
-        simResultat?.feilutbetaling < beløpsgrenseForMigreringMedFeilutbetaling &&
-        erNegativeMånedsbeløpPåMaksEnKrone(simResultat?.perioder);
+    const harStoppetMigreringAvvikInnenforBeløpsgrenser =
+        erMigreringMedStoppISimulering &&
+        simResultat &&
+        simResultat?.feilutbetaling < maksgrenseForAvvikIBeløpVedMigrering &&
+        simResultat?.etterbetaling < maksgrenseForAvvikIBeløpVedMigrering &&
```

Comment:
Skal det være < eller <=? Jeg leser favro som <=

Reply:
Er ikke helt sikker, siden det foreslåtte varselet til SB samtidig tilsier <

Reply:
Jeg har visst endret det til <= i den siste oppdatering, i hvert fall 😄 Har forresten også skrevet litt om på koden for at det skal ligne litt mer på det som er skrevet i backend etter feedback fra Halvor. Så fint om du kunne tatt en liten titt på det igjen imorgen @kristeine 🙏